.github/actions/package/action.yml

```diff
@@ -0,0 +1,32 @@
+name: 'Package rippled'
```

Comment:
If this action is only used in one place, is it simpler to just inline it?

Reply:
Having this be an action is overkill but I was just trying it out and anticipating removing this code from the repo.